db/migration/V020__insert_default_activities.sql

```diff
@@ -0,0 +1,125 @@
+INSERT INTO curriculums (principal_id, title) VALUES
+  (1, 'Professional Mentorship Program');
+
+INSERT INTO activities (title, description_text, curriculum_week, curriculum_day, start_time, end_time, duration, activity_type_id, curriculum_id) VALUES
+  -- Week 1: Day 1
+  ('Morning Standup', '', 1, 1, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Self-introduction', 'Get to know each other.', 1, 1, '11:10:00', '12:00:00', 50, 1, 1),
+  -- Week 1: Day 2
+  ('Self-assessment', '', 1, 2, NULL, NULL, NULL, 2, 1),
+  ('PMP Inroduction', 'Overview of the project and technological environment.', 1, 2, '11:00:00', '12:00:00', 60, 1, 1),
+  ('Set up Local Copy of Repository', 'Clone the repository and follow the instructions to set it up on your development machine.', 1, 2, NULL, NULL, NULL, 2, 1),  
+  -- Week 1: Day 3
+  ('Morning Standup', 'Check progress.', 1, 3, '11:00:00', '12:00:00', 60, 3, 1),
+  ('Product Brief', 'Go through the Product Brief.', 1, 3, '12:10:00', '13:00:00', 50, 1, 1),
+  -- Week 1: Day 4
+  ('Game Time', '', 1, 4, '10:00:00', '10:30:00', 30, 1, 1),
+  ('Product Specification', 'Go through the Product Specification.', 1, 4, '11:00:00', '13:00:00', 120, 1, 1),
+  -- Week 1: Day 5
+  ('Self-reflection', '', 1, 5, '10:00:00', '11:00:00', 60, 4, 1),
+  ('Data flow demostration', 'Go through the existing codes on the app.', 1, 5, '11:05:00', '13:00:00', 115, 1, 1),
+  -- Week 2: Day 1
+  ('Morning Standup', '', 2, 1, '10:00:00', '11:00:00', 60, 3, 1), 
+  ('Prototyping', 'Create messy code at the early stages of building a product to demonstrate its intended functionalities and design.', 2, 1, NULL, NULL, NULL, 1, 1),
+  -- Week 2: Day 2
+  ('Self-assessment', '', 2, 2, NULL, NULL, NULL, 2, 1),
+  ('Prototyping', 'Create messy code at the early stages of building a product to demonstrate its intended functionalities and design.', 2, 2, NULL, NULL, NULL, 1, 1),
+  -- Week 2: Day 3
+  ('Morning Standup', 'Check the progress.', 2, 3, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Prototyping', 'Create messy code at the early stages of building a product to demonstrate its intended functionalities and design.', 2, 3, NULL, NULL, NULL, 1, 1),
+  -- Week 2: Day 4
+  ('Game Time', '', 2, 4, '10:00:00', '10:30:00', 30, 1, 1),
+  ('Prototyping', 'Create messy code at the early stages of building a product to demonstrate its intended functionalities and design.', 2, 4, NULL, NULL, NULL, 1, 1),
+  -- Week 2: Day 5
+  ('Self-reflection', '', 2, 5, '10:00:00', '11:00:00', 60, 4, 1),
+  ('Prototyping Demo', 'Demostrate the prototype from each group.', 2, 5, '11:10:00', '13:30:00', 140, 2, 1),
+  -- Week 3: Day 1
+  ('Morning Standup', 'Plan the schedule for the week and assign tickets.', 3, 1, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Github Collaboration Demo', '', 3, 1, '11:10:00', '12:00:00', 50, 1, 1),
```

Comment:
Much to my chagrin, GitHub capitalizes the "H".

Reply:
Fix it in the commit: https://github.com/OpenTree-Education/rhizone-lms/pull/421/commits/3a224d805fedcf773be3ee52afb57ebe75381287